commons/src/main/java/com/simpletools/calculator/commons/operations/TaxOperation.kt

```diff
@@ -35,4 +35,8 @@ object TaxOperation {
     fun getFormula(): String {
         return "Tax in" + location + " is " + taxesMap[location]
     }
+
+    fun getTaxRate(province: String): Double {
+        return taxesMap[province]!!
```

Comment:
What is this doing ?

Reply:
It is used to display the tax rate on a Toast to the user upon pressing on the tax button.